jib task failure caused by hidden or deprecated method in jib API

In 184.2, ```jib``` task failed with the error below.
```
Caused by: java.lang.NoSuchMethodError: com.google.cloud.tools.jib.api.RegistryImage.toImageConfiguration()Lcom/google/cloud/tools/jib/configuration/ImageConfiguration;
        at org.curioswitch.gradle.golang.tasks.JibTask.exec(JibTask.java:117)
```

```toImageConfiguration()``` seems to be private  or deprecated in latest version.
https://github.com/GoogleContainerTools/jib/pull/1659/files

--- a/tools/gradle-plugins/gradle-golang-plugin/build.gradle.kts
+++ b/tools/gradle-plugins/gradle-golang-plugin/build.gradle.kts
@@ -33,8 +33,8 @@ dependencies {
     compile(project(":tools:gradle-plugins:gradle-tool-downloader-plugin"))
     compile(project(":tools:gradle-plugins:gradle-helpers"))
 
-    compile("com.google.cloud.tools:jib-gradle-plugin:0.10.2-RAG1")
     compile("com.google.guava:guava")
+    compile("gradle.plugin.com.google.cloud.tools:jib-gradle-plugin")
 
     annotationProcessor("org.immutables:value")
     compileOnly("org.immutables:value-annotations")

--- a/tools/gradle-plugins/gradle-golang-plugin/gradle.properties
+++ b/tools/gradle-plugins/gradle-golang-plugin/gradle.properties
@@ -22,4 +22,4 @@
 # SOFTWARE.
 #
 
-version = 0.0.18
+version = 0.0.19

--- a/tools/gradle-plugins/gradle-golang-plugin/src/main/java/org/curioswitch/gradle/golang/tasks/JibTask.java
+++ b/tools/gradle-plugins/gradle-golang-plugin/src/main/java/org/curioswitch/gradle/golang/tasks/JibTask.java
@@ -32,6 +32,7 @@ import com.google.cloud.tools.jib.configuration.LayerConfiguration;
 import com.google.cloud.tools.jib.configuration.Port;
 import com.google.cloud.tools.jib.filesystem.AbsoluteUnixPath;
 import com.google.cloud.tools.jib.frontend.CredentialRetrieverFactory;
+import com.google.cloud.tools.jib.image.ImageReference;
 import java.nio.file.Path;
 import org.curioswitch.gradle.golang.GolangExtension;
 import org.gradle.api.DefaultTask;
@@ -114,10 +115,10 @@ public class JibTask extends DefaultTask {
     var targetImage = RegistryImage.named(this.targetImage.get());
     if (credentialHelper.isPresent()) {
       baseImage.addCredentialRetriever(
-          CredentialRetrieverFactory.forImage(baseImage.toImageConfiguration().getImage())
+          CredentialRetrieverFactory.forImage(ImageReference.parse(this.baseImage.get()))
               .dockerCredentialHelper(credentialHelper.get()));
       targetImage.addCredentialRetriever(
-          CredentialRetrieverFactory.forImage(targetImage.toImageConfiguration().getImage())
+          CredentialRetrieverFactory.forImage(ImageReference.parse(this.targetImage.get()))
               .dockerCredentialHelper(credentialHelper.get()));
     }
 

